examples/client/getAccountInfo.ts

```diff
@@ -0,0 +1,67 @@
+import { AccountAddress, AccountInfo } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn example getAccountInfo
+
+  Options
+    --help,      -h  Displays this message
+    --endpoint,  -e  Specify endpoint of the form "address:port"
+    --account,   -a  An account address to get info from
+    --blockhash, -b  A blockhash to query the info from
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                shortFlag: 'e',
+                default: 'node.testnet.concordium.com:20000',
```

Comment:
I think these defaults should be `localhost:20000`.

We don't want people to accidentally spam our node.